components/Note/src/note.css

```diff
@@ -0,0 +1,49 @@
+.denhaag-note {
+  align-items: flex-start;
+  color: var(--denhaag-note-color);
+  display: flex;
+  flex-direction: row;
+  font-family: var(--denhaag-note-font-family, inherit);
+  font-size: var(--denhaag-note-font-size, 1.125em);
+  font-weight: var(--denhaag-note-font-weight, normal);
+  line-height: var(--denhaag-note-line-height, 1.3);
+  gap: var(--denhaag-note-icon-spacing);
+  padding-block-start: var(--denhaag-note-padding-block, calc(var(--denhaag-note-icon-spacing) / 2));
+  padding-block-end: var(--denhaag-note-padding-block, calc(var(--denhaag-note-icon-spacing) / 2));
+}
+
+.denhaag-note--info {
+  --denhaag-note-icon-color: var(--denhaag-note-icon-info-color, #ffffff);
+  --denhaag-note-icon-fill: var(--denhaag-note-icon-info-fill);
+  --denhaag-note-icon-stroke: var(--denhaag-note-icon-info-stroke, var(--denhaag-note-icon-info-fill));
+}
+
+.denhaag-note--warning {
+  --denhaag-note-icon-color: var(--denhaag-note-icon-warning-color, #ffffff);
+  --denhaag-note-icon-fill: var(--denhaag-note-icon-warning-fill);
+  --denhaag-note-icon-stroke: var(--denhaag-note-icon-warning-stroke, var(--denhaag-note-icon-warning-fill));
+}
+
+.denhaag-note__icon,
+.denhaag-note svg {
+  aspect-ratio: 1 / 1;
+  flex: 0 0 auto;
+  height: auto;
+  width: var(--denhaag-note-icon-size, calc(var(--denhaag-note-icon-spacing) * 3));
+}
+
+.denhaag-note__icon path,
+.denhaag-note svg path {
+  stroke-width: 2;
+  stroke-linecap: round;
+  stroke-linejoin: round;
+}
```

Comment:
Should we really include this CSS? What if another organisation wants to use their own icons?